### ISSUE TITLE
Unused code

### ISSUE THREAD

**Issue by avandekleut** [denisyarats/drq#13]
While modifying this code for research, I found lots of code to be redundant.
- Code for train() or eval(): only BatchNorm and DropOut are affected by this.
- Done flags in replay buffer: they are stored but never used.
- output_dim in Encoder is unused
- output_logits is always false

**Comment by denisyarats** [comment]
This open source code was just a snapshot of a larger and more general code base, where those things are being used.
